runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/tasks/AbstractDokkaTask.kt

```diff
@@ -220,7 +220,7 @@ abstract class AbstractDokkaTask : DefaultTask() {
         when (level) {
             "debug" -> logger.debug(message)
             "info" -> logger.info(message)
-            "progress" -> logger.lifecycle(message)
+            "progress" -> logger.info(message)
```

Comment:
Gradle's `lifecycle` level [(described as `Progress information messages`)](https://docs.gradle.org/current/userguide/logging.html#logLevels) is the ideal mapping option for Dokka's `progress` level, so I don't think this change is right, sorry